src/lib/core/a11y/interactivity-checker.ts

```diff
@@ -16,6 +17,8 @@ import {Injectable} from '@angular/core';
 @Injectable()
 export class InteractivityChecker {
 
+  constructor(private platform: MdPlatform) {}
```

Comment:
Can you also update the class comment to just say 
```ts
/**
 * Utility for checking the interactivity of an element, such as whether is
 * focusable or tabbable.
 */
```

And then add another `//` comment above it that explains how we drew much of this logic from ally.js, omitting checks for platforms and edge-cases we don't support?